doc/styles/config/vocabularies/ANSYS/accept.txt

```diff
@@ -91,7 +91,8 @@ reStructuredText
 REST
 RPC
 RST
-rulesets
+(?i)rulesets
+(?i)ruleset
```

Comment:
```suggestion
[Rr]ulesets
[Rr]uleset
```
Otherwise rUlEsEt would be a valid  name

Reply:
Makes sense, thanks 🙂